Gemfile.lock

```diff
@@ -616,65 +618,63 @@ GEM
     sawyer (0.9.2)
       addressable (>= 2.3.5)
       faraday (>= 0.17.3, < 3)
-    sentry-rails (5.4.2)
+    sentry-rails (5.16.1)
       railties (>= 5.0)
-      sentry-ruby (~> 5.4.2)
-    sentry-ruby (5.4.2)
+      sentry-ruby (~> 5.16.1)
+    sentry-ruby (5.16.1)
       concurrent-ruby (~> 1.0, >= 1.0.2)
     sentry-user_informer (0.2.0)
       rack
       railties (>= 6.1.0, < 7.1.0)
       sentry-rails (~> 5.4)
-    sexp_processor (4.12.1)
-    single_cov (1.3.2)
-    slop (3.6.0)
+    sexp_processor (4.17.1)
+    single_cov (1.11.0)
     snaky_hash (2.0.1)
       hashie
       version_gem (~> 1.1, >= 1.1.1)
     socksify (1.7.1)
-    soft_deletion (1.6.0)
-      activerecord (>= 4.2.0, < 6.2.0)
+    soft_deletion (1.9.0)
+      activerecord (>= 5.0.0, < 7.2)
     sprockets (3.7.2)
       concurrent-ruby (~> 1.0)
       rack (> 1, < 3)
     sprockets-rails (3.4.2)
       actionpack (>= 5.2)
       activesupport (>= 5.2)
       sprockets (>= 3.0.0)
-    sqlite3 (1.4.2)
-    stackprof (0.2.12)
+    sqlite3 (1.7.2)
+      mini_portile2 (~> 2.8.0)
+    stackprof (0.2.26)
     terminal-table (1.8.0)
       unicode-display_width (~> 1.1, >= 1.1.1)
     thor (1.3.0)
-    tilt (2.0.10)
+    tilt (2.3.0)
```

Comment:
### [tilt](https://github.com/jeremyevans/tilt)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 2.0.10 → 2.3.0

[_[change-log](https://github.com/jeremyevans/tilt/blob/master/CHANGELOG.md), [source-code](https://github.com/jeremyevans/tilt), [gem-diff](https://my.diffend.io/gems/tilt/2.0.10/2.3.0)_]





<details>
<summary>Commits</summary>

A change of **175** commits. See the full changes on [the compare page](https://github.com/jeremyevans/tilt/compare/v2.0.10...v2.3.0).

These are the first 10 commits:
- (7811a2e) [Handle rendering BasicObject instances](https://github.com/jeremyevans/tilt/commit/7811a2e8e3e2acf122960ec43c30744cd8bc12a2)
- (aea852b) [Fix #extensions_for for RedcarpetTemplate](https://github.com/jeremyevans/tilt/commit/aea852bfb90f9968f03a16d8da8574b81bca3316)
- (5805e08) [Fix markdown documentation 404](https://github.com/jeremyevans/tilt/commit/5805e0817bc77e7872896dc2634fe5dff33e81f4)
- (eb4b4ce) [Update .travis.yml](https://github.com/jeremyevans/tilt/commit/eb4b4cea3a008b7a4b6ef3faa6391503d0b0c320)
- (3169517) [GitHub is HTTPS by default](https://github.com/jeremyevans/tilt/commit/3169517f35b031d6da3675447d0487b53d40a8d8)
- (91838d5) [Fix Ruby 3.0 compatibility.](https://github.com/jeremyevans/tilt/commit/91838d57aa35ba5341368ead0d442e1dc1eb8730)
- (e7432c5) [Allow all options available in CommonMarker](https://github.com/jeremyevans/tilt/commit/e7432c54338f54c9e432282a83a6b6b3deb10f77)
- (9b02c6f) [Add Tilt::EmacsOrg support](https://github.com/jeremyevans/tilt/commit/9b02c6f27e720abb0ec3e95856c6c14df24c9b15)
- (b1329a8) [Sass: Support sass-embedded gem](https://github.com/jeremyevans/tilt/commit/b1329a8a4883f0e08b12f1e639922105d3234438)
- (911e459) [Update .travis.yml](https://github.com/jeremyevans/tilt/commit/911e459cbfa3414606b918d07e9a3b981b9e6f88)


</details>